pkg/healthcheck/healthcheck.go

```diff
@@ -489,6 +489,13 @@ func (hc *HealthChecker) allCategories() []category {
 						return hc.checkCanGet(hc.ControlPlaneNamespace, "", "v1", "secrets")
 					},
 				},
+				{
+					description: "extension API server authentication is properly configured",
```

Comment:
Okay I just read about the RBAC issue here https://linkerd.io/2017/07/24/using-linkerd-kubernetes-rbac/ . Yes it makes sense to explicitly say that the "extension-apiserver-authentication configmap" can be read instead of properly configured. I'll make the changes now